README.md

```diff
@@ -94,7 +91,7 @@ region = eu-west-1
 Download our private keys from the `membership-private` S3 bucket. If you have the AWS CLI set up you can run:
 
 ```
-aws s3 cp s3://membership-private/DEV/membership-keys.conf /etc/gu --profile membership
```

Comment:
I'd recommend using `~/.gu` instead of `/etc/gu`. This has quite a few benefits but the main one that's relevant here is that you won't need to run this command as root.